bundles/binding/org.openhab.binding.satel/src/main/java/org/openhab/binding/satel/command/IntegraStateCommand.java

```diff
@@ -72,25 +75,12 @@ protected boolean isResponseValid(SatelMessage response) {
             logger.error("Invalid response code: {}", response.getCommand());
             return false;
         }
-        if (!isPayloadLengthValid(response.getPayload().length)) {
-            logger.error("Invalid payload length for this object type {}: {}", this.stateType.getObjectType(),
+        if (response.getPayload().length != this.stateType.getPayloadLength(isExtended())) {
+            logger.error("Invalid payload length for this state type {}: {}", this.stateType,
```

Comment:
The level hasn't changed in this PR, it was always an error. But I can change it, if you wish.